### ISSUE TITLE
Default `brew install --cask thunderbird` command installs release instead of esr

### ISSUE THREAD

**Issue by mo-st** [Homebrew/homebrew-cask#191272]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [X] I have retried my command with `--force`.
- [X] I ran `brew update-reset && brew update` and retried my command.
- [X] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match / SHA256 mismatch](https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match) error (do not open an issue before trying to open a PR to fix first).

### Description of issue

If this is desired behavior, I apologize for wasting your time, but I think this should really be handled differently.

# Issue
`brew install --cask thunderbird` fetches the release version of thunderbird. I know the name is confusing if not downright misleading, but this is *not* the default or intended version of thunderbird. Instead the esr version (which is also available via `brew install --cask thunderbird@esr`, thanks for this!) should be used. Three pieces of evidence to support this claim:
1. Hitting the download button without any further specification on [thunderbird](https://www.thunderbird.net/en-US/) will download esr
2. Going into the [download options](https://www.thunderbird.net/en-US/thunderbird/all/) and selecting the non-default release channel will result in a warning window. See here: ![warning-window](https://github.com/user-attachments/assets/9a02362f-ca5e-44df-8862-bad915081eb5)
3. Downloading and running the release version will result in a huge red warning in the app, informing the user that this is an unsupported version of Thunderbird. See here: 
![unsupported-release](https://github.com/user-attachments/assets/9ddb204a-66f9-42ec-87b3-8df832e64b27)

Using this (monthly) release version could put users at risk of data loss or deteriorate user experience with thunderbird and brew. In my humble opinion this should be changed, even if the name "release" would suggest stability. Additionally, as happened to me, users may realize the mistake and install the supported release again, only to find that the profile modified by the new and unstable version is no longer compatible with the stable esr version. This forces you to set the app up again completely from scratch.

# Solution
Replace the default beaviour with the one found in `thunderbird@esr`, instead creating a new version called for example `thunderbird@release` or maybe more clearly `thunderbird@monthly`

Thank you for your time and effort, I appreciate all that you do!

### Command that failed

None

### Output of command with `--verbose --debug`

```shell
None
```

### Output of `brew doctor` and `brew config`

```shell
None
```

### Output of `brew tap`

```shell
None
```

**Comment by SMillerDev** [comment]
Duplicate of https://github.com/Homebrew/homebrew-cask/issues/187251
